docs/attributes-registry/db.md

```diff
@@ -25,7 +25,7 @@ This group defines the attributes used to describe telemetry in the context of d
 | `db.namespace`                   | string | The name of the database, fully qualified within the server address and port. [2]                                                                                                                                                                                                    | `customers`; `test.users`                                             | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
 | `db.operation.batch.size`        | int    | The number of queries included in a [batch operation](/docs/database/database-spans.md#batch-operations). [3]                                                                                                                                                                        | `2`; `3`; `4`                                                         | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
 | `db.operation.name`              | string | The name of the operation or command being executed. [4]                                                                                                                                                                                                                             | `findAndModify`; `HMSET`; `SELECT`                                    | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
-| `db.query.parameter.<key>`       | string | The query parameters used in `db.query.text`, with `<key>` being the parameter name, and the attribute value being the parameter value. [5]                                                                                                                                          | `someval`; `55`                                                       | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `db.query.parameter.<key>`       | string | A query parameter used in `db.query.text`, with `<key>` being the parameter name, and the attribute value being a string representation of the parameter value. [5]                                                                                                                  | `someval`; `55`                                                       | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```

Comment:
https://github.com/open-telemetry/semantic-conventions/issues/1157 would be helpful here as we'd then have `"55"` instead